client/my-sites/email/email-forwarding/email-forwarding-add-new-compact-list.jsx

```diff
@@ -0,0 +1,173 @@
+import { Button, Gridicon } from '@automattic/components';
+import { localize } from 'i18n-calypso';
+import page from 'page';
+import PropTypes from 'prop-types';
+import { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+import { validateAllFields } from 'calypso/lib/domains/email-forwarding';
+import EmailForwardingAddNewCompact from 'calypso/my-sites/email/email-forwarding/email-forwarding-add-new-compact';
+import { emailManagement } from 'calypso/my-sites/email/paths';
+import {
+	composeAnalytics,
+	recordTracksEvent,
+	withAnalytics,
+} from 'calypso/state/analytics/actions';
+import { addEmailForward } from 'calypso/state/email-forwarding/actions';
+import getEmailForwardingLimit from 'calypso/state/selectors/get-email-forwarding-limit';
+import getEmailForwardingType from 'calypso/state/selectors/get-email-forwarding-type';
+import { getEmailForwards, isAddingForward } from 'calypso/state/selectors/get-email-forwards';
+import { getSiteSlug } from 'calypso/state/sites/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+
+class EmailForwardingAddNewCompactList extends Component {
+	static propTypes = {
+		emailForwards: PropTypes.array,
+		emailForwardingLimit: PropTypes.number,
```

Comment:
I think we should stick with old name, just because there would be an incosistence between the getter (which is `getEmailForwardingLimit` ) and variable name. Don't you think so?

Reply:
That's a good point indeed.